hazelcast/src/main/java/com/hazelcast/nio/tcp/nonblocking/NonBlockingIOThread.java

```diff
@@ -171,10 +190,15 @@ public final void run() {
         try {
             for (; ; ) {
                 try {
-                    if (selectNow) {
-                        runSelectNowLoop();
-                    } else {
-                        runSelectLoop();
+                    switch (selectorMode) {
```

Comment:
I would rename this to select mode. Because it says something about which select approach is being used; not the selector.